l1-contracts/contracts/state-transition/chain-deps/facets/Mailbox.sol

```diff
@@ -317,7 +316,7 @@ contract MailboxFacet is ZKChainBase, IMailboxImpl, MessageVerification {
             revert NotHyperchain();
         }
         /// We pause L1->GW->L2 deposits.
-        require(_checkV30UpgradeProcessed(_chainId), DepositsPaused());
+        require(_checkV30UpgradeProcessed(_chainId) && !_depositsPaused(), DepositsPaused());
```

Comment:
actually this modification is not needed, since if the v30 upgrade is processed, then deposits paused check is done already in the chain's Mailbox